### ISSUE TITLE
Session Start Error

### ISSUE THREAD

**Issue by MelusiZondi** [SafeExamBrowser/seb-win-refactoring#986]
> [!IMPORTANT]
> - Please _always_ consult the documentation first before creating a bug report: https://safeexambrowser.org/windows/win_usermanual_en.html.
> - Please _always_ attach the log file(s) of the affected session(s)! They can be found under `%LocalAppData%\SafeExamBrowser\Logs`.

[2024-10-04_07h43m46s_Browser.log](https://github.com/user-attachments/files/17254408/2024-10-04_07h43m46s_Browser.log)
[2024-10-04_07h43m46s_Client.log](https://github.com/user-attachments/files/17254409/2024-10-04_07h43m46s_Client.log)
[2024-10-04_07h43m46s_Runtime.log](https://github.com/user-attachments/files/17254410/2024-10-04_07h43m46s_Runtime.log)

**Describe the Bug**
SEB failed to start a new session

**Steps to Reproduce**

**Expected Behavior**
SEB launches then student attempts the assessment.
**Screenshots**
![image](https://github.com/user-attachments/assets/970a7177-deea-4868-9825-c3efd3980628)

**Version Information**
 - OS: [e.g. Windows 10 Professional, Version 1803]
 - SEB-Version [e.g. SEB 3.0.1]

**Additional Context**
This error has been occurring across several devices.

**Comment by dbuechel** [comment]
Again, the reason can be found in plain English in the runtime application log:

```
2024-10-04 07:44:11.306 [25] - INFO: Validating display configuration...
2024-10-04 07:44:11.319 [25] - INFO: [DisplayMonitor] Detected active, external display 'DISPLAY\HWP326F\4&266e0cb&0&UID4145_0' connected via 'HDMI'.
2024-10-04 07:44:11.322 [25] - INFO: [DisplayMonitor] Detected active, internal display 'DISPLAY\AUO1F92\4&266e0cb&0&UID8388688_0' connected via 'Internal'.
2024-10-04 07:44:11.323 [25] - WARNING: [DisplayMonitor] Detected 2 active displays but only 1 are allowed!
2024-10-04 07:44:11.338 [25] - ERROR: Display configuration is not allowed!
```
